man/sd_bus_message_new_method_error.xml

```diff
@@ -0,0 +1,184 @@
+-->
+
+<refentry id="sd_bus_message_new_method_error"
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>sd_bus_message_new_method_error</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>sd_bus_message_new_method_error</refentrytitle>
+    <manvolnum>3</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>sd_bus_message_new_method_error</refname>
+    <refname>sd_bus_message_new_method_errorf</refname>
+    <refname>sd_bus_message_new_method_errno</refname>
+    <refname>sd_bus_message_new_method_errnof</refname>
+
+    <refpurpose>Create a an error reply for a method call</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <funcsynopsis>
+      <funcsynopsisinfo>#include &lt;systemd/sd-bus.h&gt;</funcsynopsisinfo>
+
+      <funcprototype>
+        <funcdef>int sd_bus_message_new_method_error</funcdef>
+        <paramdef>sd_bus_message *<parameter>call</parameter></paramdef>
+        <paramdef>sd_bus_message **<parameter>m</parameter></paramdef>
+        <paramdef>const sd_bus_error *<parameter>e</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int sd_bus_message_new_method_errorf</funcdef>
+        <paramdef>sd_bus_message *<parameter>call</parameter></paramdef>
+        <paramdef>sd_bus_message **<parameter>m</parameter></paramdef>
+        <paramdef>const char *<parameter>name</parameter></paramdef>
+        <paramdef>const char *<parameter>format</parameter></paramdef>
+        <paramdef>...</paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int sd_bus_message_new_method_errno</funcdef>
+        <paramdef>sd_bus_message *<parameter>call</parameter></paramdef>
+        <paramdef>sd_bus_message **<parameter>m</parameter></paramdef>
+        <paramdef>int <parameter>error</parameter></paramdef>
+        <paramdef>const sd_bus_error *<parameter>p</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int sd_bus_message_new_method_errnof</funcdef>
+        <paramdef>sd_bus_message *<parameter>call</parameter></paramdef>
+        <paramdef>sd_bus_message **<parameter>m</parameter></paramdef>
+        <paramdef>int <parameter>error</parameter></paramdef>
+        <paramdef>const char *<parameter>format</parameter></paramdef>
+        <paramdef>...</paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para>The <function>sd_bus_message_new_method_error()</function> function creates
+    a new message that is an error reply to the <parameter>call</parameter> message,
+    and returns it in the <parameter>m</parameter> output parameter. The error
+    information from error <parameter>e</parameter> is appended: the
+    <parameter>name</parameter> field of <parameter>e</parameter> is used as the
+    "name" field in the reply header, and and the <parameter>message</parameter>
```

Comment:
"name" field → error identifier?

maybe also say: 

```
(for example and error name such as <literal>org.freedesktop.DBus.Error.NotSupported</literal> or the equivalent symboligc <constant>SD_BUS_ERROR_NOT_SUPPORTED</constant>)
```

simply to illustrate what kind of parameter is requested here.